auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/EmailService.kt

```diff
@@ -177,11 +185,17 @@ class EmailService(
         logger.atInfo().addKeyValue("userCN", userCN).log("Sent password-never-set email")
     }
 
-    suspend fun sendResetPasswordEmail(user: LdapUser, token: String, call: ApplicationCall) {
+    suspend fun sendResetPasswordEmail(
+        user: LdapUser,
+        token: String,
+        triggeredByAdmin: Boolean,
```

Comment:
nitpick, don't need to change it, but would be nicer to have it be e.g. `triggeringAdminSession: OAuthSession?` or similar. It's a bit more faff, but it makes it much harder to pass the wrong thing in, means sending an email doesn't need to know about the principal in the application call, and is clearer to read (hard to know what true means in `sendResetPasswordEmail(user, token, true, call)` without reading the function. Generally passing optional thing in > enum > boolean flag).